frontend/src/pages/ImportCkan/SelectGroupsScreen.jsx

```diff
@@ -0,0 +1,589 @@
+        }
+        await fetch('/api/Metadata/url', {
+          method: 'PUT',
+          body: JSON.stringify(payload),
+          headers: {
+            'Content-Type': 'application/json',
+            Authorization: `bearer ${token}`,
+          },
+        });
+      }
+    }
+    /* eslint-enable no-await-in-loop */
+    await LoadMetadataTypes();
+    nextPage();
+    setSaving(false);
+  };
+
+  const setInsertStrategy = (strategy) => {
+    const grp = currentGroup;
+    grp.openDataMetadata.strategy = strategy;
+    updateGroup(currentIndex, grp);
+  };
+
+  const handleMetadataTypeChange = (v) => {
+    // console.log(v);
+    const grp = currentGroup;
+    grp.openDataMetadata.metadataType = v.target.value;
+    updateGroup(currentIndex, grp);
+  };
+
+  const handleMetadataChange = (v) => {
+    const grp = currentGroup;
+    grp.openDataMetadata.metadata = v.target.value;
+    updateGroup(currentIndex, grp);
+  };
+
+  const updateName = (n) => {
+    const grp = currentGroup;
+    grp.openDataMetadata.metadataTypeName = n;
+    updateGroup(currentIndex, grp);
+  };
+
+  const updateDescription = (d) => {
+    const grp = currentGroup;
+    grp.openDataMetadata.metadataTypeDescription = d;
+    updateGroup(currentIndex, grp);
+  };
+
+  const onCategoryChange = (c) => {
+    const categoryUuid = c.currentTarget.value;
+
+    const grp = currentGroup;
+    grp.openDataMetadata.metadataTypeCategory = categoryUuid;
+    updateGroup(currentIndex, grp);
+  };
+
+  const updateResourceName = (index, value) => {
+    const grp = currentGroup;
+    grp.resources[index].name = value;
+    updateGroup(currentIndex, grp);
+  };
+
+  const handleResourceMetadataChange = (index, name, value) => {
+    const grp = currentGroup;
+    grp.resources[index].openDataMetadata[name] = value;
+    updateGroup(currentIndex, grp);
+  };
+
+  let conversionStrategyGui = (
+    <div>
+      <h2>Create new metadata type</h2>
+      <Input type="text" value={currentGroup.openDataMetadata.metadataTypeName} onChange={(e) => updateName(e.target.value)} placeholder="Name" required />
+      <TextArea value={currentGroup.openDataMetadata.metadataTypeDescription} onChange={(e) => updateDescription(e.target.value)} placeholder="Description" required />
+      <RootLevelCategorySelector onChange={onCategoryChange} />
+    </div>
+  );
+
+  if (currentGroup.openDataMetadata.strategy === 1) {
+    conversionStrategyGui = (
+      <div>
+        <h2>Select metadata type</h2>
+        <Select name="metadataTypeName" value={currentGroup.openDataMetadata.metadataType} onChange={handleMetadataTypeChange} required>
+          <option value="" disabled>Metadata type</option>
+          {
+            Object.keys(metadataTypes).map((key) => (
+              <option key={key} value={key}>{metadataTypes[key].name}</option>
+            ))
+          }
+        </Select>
+      </div>
+    );
+  } else if (currentGroup.openDataMetadata.strategy === 1) {
```

Comment:
Isn't else if the exact same statement as in the original if? I.e. will it ever trigger? If not, is it intended to?

Reply:
oopsie whoopsie